### ISSUE TITLE
2018-02-16 Public User Feedback Meeting - Node.js Benchmarking

### ISSUE THREAD

**Issue by dshaw** [nodejs/user-feedback#27]
# 2018-02-16 Public User Feedback Meeting - Node.js Benchmarking

## Time

Friday, 2018-02-16 at 17:00 UTC / noon ET / 11am CT / 9am PT

## Links

* **Minutes Doc**: https://docs.google.com/document/d/1RoPPTUxR9xtZ_3NS4Z5P8mXHZrKyGz3jwGPnD9hpjlA/edit
* **Previous Minutes Doc**: https://docs.google.com/document/d/13kjBLpK3huHgCebxROpBNuUQLNyyOafaI5lOKi129Co/edit

## Agenda

This is a public User Feedback meeting with [invited participants](https://github.com/nodejs/user-feedback/issues/20) focused on specific topics. Please feel free to propose discussion items for the agenda related to this topic below. Create a new issue to propose a new topic. Normally, @nodejs/user-feedback [team meetings](https://github.com/nodejs/user-feedback/labels/neuf-meeting) are extracted from [neuf-agenda](https://github.com/nodejs/user-feedback/labels/neuf-agenda) labelled issues and pull requests from the [user-feedback](https://github.com/nodejs/user-feedback) repo prior to the meeting.

* Introduction to the Benchmarking Working Group (WG) with @mhdawson
* Presentation of the Benchmarking Survey Results
* Detailed feedback from select attending representatives
* Open feedback forum

## Invited
* User Feedback team members
  * Dan Shaw (@dshaw - User Feedback, CommComm observer)
  * Mike Hostetler (@mikehostetler)
  * Mihai Ene-Pietrosanu (@mihaiep, User Feedback)
  * William Kapke (@williamkapke - CommComm, Individual Member Director)
  * Michael Dawson (@mhdawson - User Feedback, CommComm, TSC Chair)
  * Tierney Cyren (@bnb - User Feedback, CommComm Co-Chair)
  * Tracy Hinds (@hackygolucky - CommComm)
  * Greg Wallace (@gtewallace - Node.js Foundation)

* End User Representatives
  * Bradley Farias (@bmeck - GoDaddy)
  * Mohammad  Khan 
  * Tierney Cyren (Node Source)
  * Amanda Blackburn (Profound Logic)
  * Mittra, Partha (whisolutions)
  * Heckmann, Aaron (Paypal)
  * Aaron Bartell (Krengel Tech) 
  * Matvienko Nikolay (Grid Dynamics)
  * Jeremy Meyer (Jack Henry)
  * Bradley Meck (GoDaddy)

## Public participation

The Node.js project live streams meetings so anyone can tune in. Everyone is invited to participate in the live chat. The stream will start playing at https://www.youtube.com/c/nodejs+foundation/live once we go live.

There's usually a short cat-herding time at the start of the meeting and occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and we'll be online shortly. ❤️

## Joining the meeting

Join link for invited members and observers:  https://zoom.us/j/665550923

There is a standing invitation for all @nodejs/TSC and @nodejs/community-committee members to attend in observer capacity.

**Comment by mhdawson** [comment]
@dshaw the title does not seem to match up with the contents of the issue ?

**Comment by dshaw** [comment]
@mhdawson Fixed title. Thanks.

**Comment by dshaw** [comment]
I updated the agenda. @nodejs/user-feedback, please review.

**Comment by dshaw** [comment]
At the [2018-01-12 User Feedback team meeting](https://github.com/nodejs/user-feedback/issues/26), we decided to expand the constituency of this first public session. We had originally scheduled another broader public feedback session prior to this which had to be rescheduled. The enterprise focus was too limiting to effectively serve the folks that have raised their hands during [call for participation](https://github.com/nodejs/user-feedback/issues/20).

We will close the [Benchmarking WG Survey](https://github.com/nodejs/user-feedback/issues/22) at the end of January. Therefore, we decided to anchor this meeting topic on Benchmarking.

**Comment by mhdawson** [comment]
@dshaw was I was suggesting was not that we discuss the results of the benchmarking survey as we've not yet shared with the benchmarking WG (which we should do first) and we still need to figure out what can be made public.

Instead was suggesting we use the session to do discuss the questions in person as another way to gather the data we were looking for.  This will likely give us additional insights as we can have a conversation about the questions.  We should likely invite participants from the benchmark WG to attend as well.

So I think your description would be amended to be:

* Initial welcome and introductions (since this is the first time we are meeting)
* Discuss proposed approach for getting together on a regular basis
* Introduction to the Benchmarking Working Group (WG) with @mhdawson
* Benchmarking survey walkthrough, discuss experiences of users in attendance and deep dive
   into questions of most interest
* Open feedback forum

**Comment by dshaw** [comment]
@nodejs/benchmarking Please let us know if you're planning on attending and we'll add you to the agenda.

**Comment by bmeck** [comment]
I'll be attending, slowly getting data on my end.

**Comment by dshaw** [comment]
Heads-up: after a discussion with @mhdawson, there are some folks who we have not been able to invite, so we will be delaying this session by a week. We will reconvene on Friday, 2/16 at 9am PT.

🙏 Thank you 

/cc @nodejs/user-feedback @nodejs/community-committee @bradleyfarias @joelchen

**Comment by mhdawson** [comment]
Thanks for everybody's patience we want to make sure we get invites out to everybody who has volunteered before the first meeting.

**Comment by bnb** [comment]
I should be able to make this meeting 👍

**Comment by mhdawson** [comment]
Invites were sent last week, looking forward to the meeting on Friday.

**Comment by bnb** [comment]
Updated link: https://zoom.us/j/665550923

**Comment by mihaiep** [comment]
Excellent meeting. Thank you Dan & Michael for organizing and thanks all for participating and valuable feedback. https://benchmarking.nodejs.org/ (more links in meeting notes)

**Comment by bnb** [comment]
Closing, since this is a past meeting.
